server_admin/topics/users/proc-searching-user.adoc

```diff
@@ -11,7 +11,7 @@ Search for a user to view detailed information about the user, such as the user'
 * You are in the realm where the user exists.
 
 .Procedure
-. Click *Users* in the left menu. This opens the user list page.  
+. Click *Users* in the main menu. This opens the user list page.  
```

Comment:
Should be "The user list page is displayed."  to be consistent with our terminology.